src/mtev_cluster.c

```diff
@@ -65,13 +82,16 @@ struct mtev_cluster_t {
   mtev_cht_t **cht;
   mtev_net_heartbeat_ctx *hbctx;
   mtev_cluster_node_t *oldest_node;
+  struct jl_array_list *hb_payloads;
```

Comment:
The only JS specific line I see in mtev_arraylist is `mtev_json_max`. So I don't see how these array lists are not generic enough to be used for this purpose.
Using a C array would mean to implement exactly what's currently in `jl_array_list`
I didn't want to use an mtev_hash because I thought it could become a performance issue.